Add support for preferred arn per app

narkoleptik opened the pull request:
Adding support for specifying an arn to use per "app" so that you don't have to always choose from the list.

Configuration change to `clisso.yaml`:
```
apps:
  prod:
    app-id: "123456"
    principal-arn: arn:aws:iam::XXXXXXXXXXXX:policy/OneLoginListRoles
    provider: onelogin-provider
    role-arn: arn:aws:iam::XXXXXXXXXXXX:saml-provider/OneLogin
    duration: 28800
    arn: arn:aws:iam::XXXXXXXXXXXX:role/SRE
```

allcloud-jonathan commented:
Hey @narkoleptik,

thanks a lot for the PR. Since the change needed for Okta support is going to be just a few lines, could you please align them?

I'd appreciate it if you can add some information to the README file, so people understand how to use the feature.

narkoleptik commented:
@allcloud-jonathan Ok, I added in the passing of the pArn value to Okta support even though it is not used. I also added in a create option with an example shown in the readme. Let me know if there is more you want me to do, or if you have suggestions for making it better. Thanks for letting me contribute <3
